docs/release_notes.rst

```diff
@@ -2,6 +2,61 @@
 Release Notes
 ##########################################
 
+Upcoming Release
+================
+
+* **WARNING**: For some technologies the units used were changed. Check for correct usage in automatic workflows.
```

Comment:
Also add a warning about changed name from "Haber-Bosch synthesis" to "Haber-Bosch"